galata/test/jupyterlab/general.test.ts

```diff
@@ -53,4 +53,22 @@ test.describe('General Tests', () => {
     await page.reload();
     expect(await page.theme.getTheme()).toEqual('JupyterLab Dark');
   });
+
+  test('Browser tab name updates to file name in /doc mode', async ({
+    page,
+    baseURL
+  }) => {
+    // Switch to single-document mode
+    const currentUrl = page.url();
+    const docUrl = currentUrl.replace('/lab/', '/doc/');
+    await page.goto(docUrl);
+
+    // Create a new text file and wait for it to lodd
+    await page.menu.clickMenuItem('File>New>Text File');
+    await page.locator('.cm-editor:not(.jp-mod-readOnly)').waitFor();
```

Comment:
Would something like this work?

```suggestion
    await page.getByLabel('untitled.txt').locator('.cm-editor').waitFor();
```


Just guessing based on the error message